7thHeaven.Code/RegistryHelper.cs

```diff
@@ -435,6 +435,13 @@ public static void BeginTransaction()
 
         public static void CommitTransaction()
         {
+
+            string path = Sys.PathToTempFolder;
+            if (!Directory.Exists(path))
+            {
+                Directory.CreateDirectory(path);
+            }

```

Comment:
Can you please remove the `path` variable and just use directly `Sys.PathToTempFolder` instead? Thank you!

Reply:
sure, give me a sek

Reply:
Thanks :)